Task 977-984: Catalonia mqpr opus_book re_dial

zaidzidane opened the pull request.

swarooprm commented:
@nrjvarshney, wondering if you can help review this PR?

nrjvarshney commented:
General comments:
* Categories field should be a list. Please refer to the format. 
* Pass the task definition through Grammarly. 
* Use punctuations wherever necessary.
* Humans should be able to solve the task.

Task 001 - 
* Please simplify the definition. It is difficult to follow.  
* Give examples of dissimilar also 
* Why is "Label id 1 denotes similar and 0 denotes dissimilar." needed in the definition
* Write the possible outputs in the definition in quotes

Task 002 - 
* I don't think this task makes sense. Even a human can't answer it.

Task 003 
* Convert the outputs to lower case 
* Suggestion: "In this task, you are given a Spanish and a Catalan tweet and you need to detect the stance towards the target '..' . Your output should be one of 'favor', 'against', and 'neutral'".
* Add more details about this task in the definition if I missed


Task 004,5, 7, 8,9, 10 - 
* Task definition 
* I could not even understand the task.
* Examples are different from the instances.

Task 006 -
* What are these " @165710 "
* Not making sense

swarooprm commented:
@zaidzidane, can you address reviewer comments so that I can evaluate your HW?

swarooprm commented:
> @zaidzidane, can you address reviewer comments so that I can evaluate your HW?

A gentle reminder! I need to grade HW soon.

zaidzidane commented:
Sure. I will get it done by tomorrow. I hope that's ok.

zaidzidane commented:
General comments:

    Categories field should be a list. Please refer to the format.

     **I have added everything in list for categories.**
    Pass the task definition through Grammarly.
    **I did it now.**
    Use punctuations wherever necessary.
    Humans should be able to solve the task.

Task 001 -

    Please simplify the definition. It is difficult to follow.
    Give examples of dissimilar also
    Why is "Label id 1 denotes similar and 0 denotes dissimilar." needed in the definition
    Write the possible outputs in the definition in quotes
   
    **I have made the definition simpler now.**

Task 002 -

    I don't think this task makes sense. Even a human can't answer it.
 
   **I have removed this task.**


Task 003

    Convert the outputs to lower case
    Suggestion: "In this task, you are given a Spanish and a Catalan tweet and you need to detect the stance towards the target '..' . Your output should be one of 'favor', 'against', and 'neutral'".
    Add more details about this task in the definition if I missed
   
  **I have converted the output in lowercase.**

Task 004,5, 7, 8,9, 10 -

    Task definition
    I could not even understand the task.
    Examples are different from the instances.
   
   **All these task are translation task from several languages to English. I have updated the definition also.**

Task 006 -

    What are these " @165710 "
    Not making sense
  
   **These were movies names I have removed them and have placed movie names now.**

swarooprm commented:
@nrjvarshney check if you are happy or have any other feedback?

nrjvarshney commented:
* For input language , output language refer to https://github.com/allenai/natural-instructions-expansion/issues/113 
* Elaborate the explanation. Just "The translation is incorrect." is not enough.

zaidzidane commented:
For input language , output language refer to
Language indicator key/values to the schema  allenai/natural-instructions-expansion#113
**Is this for the deleted task 2?**

Elaborate the explanation. Just "The translation is incorrect." is not enough.

**I have followed the allenai nautral-instruction-expansion link as shared by Swaroop.**
https://github.com/allenai/natural-instructions-expansion/blob/master/tasks/task272_europarl_translation.json

.For the entity matching in translation I have followed this link in the alllenai instruction list. 
https://github.com/allenai/natural-instructions-expansion/blob/master/tasks/task254_spl_translation_fi_en.json

zaidzidane commented:
@swarooprm @nrjvarshney  Any comments on the feedback. 
I have my NLP mid-sem this week can I get the update early.

swarooprm commented:
Your HW2 is complete now.

Would you like to submit the same tasks to our big project repo https://github.com/allenai/natural-instructions-expansion/pulls? You just need to fork that repo, rename tasks starting from the latest number in the repo and create PR there. This way, you can be part of our research on "learning from instructions instead of learning from examples (conventional machine learning)". This is purely optional and your HW2 grade does not depend on this @zaidzidane
